themes/src/main/resources/theme/keycloak.v2/account/resources/public/patternfly-addons.css

```diff
@@ -0,0 +1,9006 @@
+.pf-u-screen-reader {
```

Comment:
This seems like a style sheet that should be loaded from the PatternFly package, can we load this in from there?

Reply:
Yes, I moved this into the pf copy scripts and removed the file.